File: API/Entities/Keyword.cs
using System.ComponentModel.DataAnnotations;

namespace GRT.Entities
{
    public class Keyword
    {
        public int Id { get; set; }
        [Required]
        public string KeywordName { get; set; }
        public string GoogleHost { get; set; }
        public string Language { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public int ProjectId { get; set; }
        public virtual Project Project { get; set; }
    }
}

File: client/src/app/_models/keyword.ts
export interface Keyword {
    id: number,
    keywordName: string,
    googleHost: string,
    language: string,
    country: string,
    city: string,
    lastPosition: any,
    lastSearch: Date,
    projectId: number,
    // project: {
    //     id: number,
    //     projectName: string,
    //     domain: string,
    //     dateAdded: Date
    // }
}